gm4_relocators/data/gm4_relocators/functions/place_down/summon_aec.mcfunction

```diff
@@ -0,0 +1,24 @@
+# @s = player who placed a relocated block player head
+# run from advancement "place_relocated_block"
+
+advancement revoke @s only gm4_relocators:place_relocated_block
+
+# store item into storage
+data modify storage gm4_relocators:data Item set from entity @s SelectedItem
+
+# check if it's a hopper or dropper to be placed down
+scoreboard players set hopper gm4_rl_data 1
+scoreboard players set dropper gm4_rl_data 1
+data modify storage gm4_relocators:data DropperCheck set value "dropper"
+data modify storage gm4_relocators:data HopperCheck set value "hopper"
+execute store result score dropper gm4_rl_data run data modify storage gm4_relocators:data DropperCheck set from storage gm4_relocators:data Item.tag.gm4_relocation_data.type
+execute store result score hopper gm4_rl_data run data modify storage gm4_relocators:data HopperCheck set from storage gm4_relocators:data Item.tag.gm4_relocation_data.type
```

Comment:
Why is all this needed?

Reply:
This code is used to set block either a hopper or a dropper. I have an idea to make this better.